### DISCUSSION TITLE
Change port for starting code server other than 3000?

### DISCUSSION THREAD

**Discussion by TheLinuxGuy-ssh** [coder/coder#11352] [General] [answered]

**Comment by matifali** [comment, marked as answer]
Hi, @rohansharma-developer, Are you talking about [`coder server`](https://coder.com/docs/v2/latest/cli/server#--http-address) or [`code-server`](https://github.com/coder/code-server)?

For Coder you can use 

```shell
coder server --http-address 127.0.0.1:PORT
```
or set the environment variable `CODER_HTTP_ADDRESS` to the same value.
